### PR TITLE
My jobs component creation

### PR DESCRIPTION
**Pull request by Ekman1991**

### CODE REVIEW

**Comment by buren** [comment]
Hello, @Ekman1991! This is your first Pull Request that will be reviewed by [Ebert](https://ebertapp.io/), an automatic Code Review service. It will leave comments on this diff with potential issues and style violations found in the code as you push new commits. You can also see all the issues found on this Pull Request on its [review page](https://ebertapp.io/github/justarrived/just-match-web/pulls/365). Please check our [documentation](https://ebertapp.io/docs) for more information.

File: src/app/components/user-job-card/user-job-card.component.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  .home-job-item-salary-will-perform-job-container {
+    background: $color-primary-cerise;
+    color: $white;
+  }
+
+  .home-job-item-salary-job-ended-container {
+    background: $color-gray-50;
+    color: $color-gray-200;
+  }
+}
```
Comment (buren) [inline src/app/components/user-job-card/user-job-card.component.scss:52]: Rule declaration should be followed by an empty line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#emptylinebetweenblocks)

File: src/app/components/user-job-card/user-job-card.component.scss [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    text-align: center;
+  }
+
+  .home-job-item-salary-will-perform-job-container {
+    background: $color-primary-cerise;
+    color: $white;
+  }
+
+  .home-job-item-salary-job-ended-container {
+    background: $color-gray-50;
```
Comment (buren) [inline src/app/components/user-job-card/user-job-card.component.scss:49]: Properties should be ordered color, background

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: src/app/components/user-job-card/user-job-card.component.scss [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    background: $color-primary-cerise;
+    padding: 10px;
+    font-weight: 500;
+    /* TODO: Jobs border radius variable */
+    border-radius: 0 0 $job-card-border-radius $job-card-border-radius;
+    text-align: center;
+  }
+
+  .home-job-item-salary-will-perform-job-container {
+    background: $color-primary-cerise;
```
Comment (buren) [inline src/app/components/user-job-card/user-job-card.component.scss:44]: Properties should be ordered color, background

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: src/app/components/user-job-card/user-job-card.component.scss [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+      @include truncate-owerflowing-text;
+    }
+  }
+
+  .home-job-item-salary-container {
+    color: $white;
+    background: $color-primary-cerise;
+    padding: 10px;
+    font-weight: 500;
+    /* TODO: Jobs border radius variable */
```
Comment (buren) [inline src/app/components/user-job-card/user-job-card.component.scss:38]: Use `//` comments everywhere

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#comment)

File: src/app/components/user-job-card/user-job-card.component.scss [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+    .home-job-item-name {
+      text-align: center;
+      white-space: normal;
+      @include truncate-owerflowing-text;
+    }
+  }
+
+  .home-job-item-salary-container {
+    color: $white;
```
Comment (buren) [inline src/app/components/user-job-card/user-job-card.component.scss:34]: Properties should be ordered padding, font-weight, color, text-align, background, border-radius

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: src/app/components/user-job-card/user-job-card.component.scss [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    height: 110px;
+    padding: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    .home-job-item-name {
+      text-align: center;
+      white-space: normal;
+      @include truncate-owerflowing-text;
```
Comment (buren) [inline src/app/components/user-job-card/user-job-card.component.scss:29]: Expected item on line 29 to appear before line 27. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#declarationorder)

File: src/app/components/user-job-card/user-job-card.component.scss [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+  border-radius: $job-card-border-radius;
+  box-shadow: 0 2px 6px 0 rgba(0,0,0,0.4);
+  cursor: pointer;
+
+  &:hover {
+    transform: scale(1.02);
+  }
+
+  .home-job-item-name-container {
+    height: 110px;
```
Comment (buren) [inline src/app/components/user-job-card/user-job-card.component.scss:20]: Properties should be ordered display, height, padding, flex-direction

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: src/app/components/user-job-card/user-job-card.component.scss [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+@import '../../styles/typography';
+$job-card-border-radius: 8px;
+
+.home-job-item-container {
+  display: inline-block;
+  width: 180px;
+  margin: 5px;
+  background-color: $default-bg-color;
+  border-radius: $job-card-border-radius;
+  box-shadow: 0 2px 6px 0 rgba(0,0,0,0.4);
```
Comment (buren) [inline src/app/components/user-job-card/user-job-card.component.scss]: `0.4` should be written without a leading zero as `.4`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)

Comment (buren) [inline src/app/components/user-job-card/user-job-card.component.scss]: Commas in function arguments should be followed by a single space

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#spaceaftercomma)

Comment (buren) [inline src/app/components/user-job-card/user-job-card.component.scss]: Commas in function arguments should be followed by a single space

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#spaceaftercomma)

Comment (buren) [inline src/app/components/user-job-card/user-job-card.component.scss]: Commas in function arguments should be followed by a single space

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#spaceaftercomma)

File: src/app/components/user-job-card/user-job-card.component.scss [hunk 9]
```diff
@@ -0,0 +1,61 @@
+@import '../../styles/colors';
+@import '../../styles/dimensions';
+@import '../../styles/typography';
+$job-card-border-radius: 8px;
+
+.home-job-item-container {
+  display: inline-block;
+  width: 180px;
+  margin: 5px;
+  background-color: $default-bg-color;
```
Comment (buren) [inline src/app/components/user-job-card/user-job-card.component.scss:10]: Properties should be ordered display, width, margin, cursor, background-color, border-radius, box-shadow

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: src/app/components/jobs-slider/jobs-slider.component.scss [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+      padding-left: 48px;
+
+      .home-jobs-applied-for-job-item-name {
+        margin-top: 15px;
+        margin-bottom: 15px;
+        @include truncate-owerflowing-text;
+      }
+
+      .home-jobs-applied-for-job-item-icon {
+        color: $color-primary-cerise;
```
Comment (buren) [inline src/app/components/jobs-slider/jobs-slider.component.scss:100]: Properties should be ordered padding-left, margin-top, margin-right, margin-bottom, margin-left, color

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: src/app/components/jobs-slider/jobs-slider.component.scss [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+    .home-jobs-applied-for-job-item {
+      cursor: pointer;
+      display: flex;
+      width: 100%;
+      padding-left: 48px;
+
+      .home-jobs-applied-for-job-item-name {
+        margin-top: 15px;
+        margin-bottom: 15px;
+        @include truncate-owerflowing-text;
```
Comment (buren) [inline src/app/components/jobs-slider/jobs-slider.component.scss:96]: Expected item on line 96 to appear before line 94. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#declarationorder)

File: src/app/components/jobs-slider/jobs-slider.component.scss [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+    border-top: 1px solid $color-gray-25;
+    border-bottom: 1px solid $color-gray-25;
+    width: 100%;
+
+    &:hover {
+      opacity: 0.5;
+    }
+
+    .home-jobs-applied-for-job-item {
+      cursor: pointer;
```
Comment (buren) [inline src/app/components/jobs-slider/jobs-slider.component.scss:88]: Properties should be ordered display, width, padding-left, cursor

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: src/app/components/jobs-slider/jobs-slider.component.scss [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+    }
+  }
+
+  .home-jobs-applied-for-job-item-container {
+    border-top: 1px solid $color-gray-25;
+    border-bottom: 1px solid $color-gray-25;
+    width: 100%;
+
+    &:hover {
+      opacity: 0.5;
```
Comment (buren) [inline src/app/components/jobs-slider/jobs-slider.component.scss:84]: `0.5` should be written without a leading zero as `.5`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)

File: src/app/components/jobs-slider/jobs-slider.component.scss [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+
+    .home-jobs-applied-for-header {
+      margin-top: 15px;
+      margin-bottom: 15px;
+      margin-left: 48px;
+    }
+  }
+
+  .home-jobs-applied-for-job-item-container {
+    border-top: 1px solid $color-gray-25;
```
Comment (buren) [inline src/app/components/jobs-slider/jobs-slider.component.scss:79]: Properties should be ordered width, border-top, border-bottom

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: src/app/components/jobs-slider/jobs-slider.component.scss [hunk 15]
```diff
@@ -0,0 +1,109 @@
...
+.home-jobs-slider-user-jobs-container {
+  background: $color-secondary-gray;
+}
+
+.home-jobs-applied-for-container {
+  display: flex;
+  flex-wrap: wrap;
+
+  .home-jobs-applied-for-header-container {
+    background: $color-gray-25;
```
Comment (buren) [inline src/app/components/jobs-slider/jobs-slider.component.scss:67]: Properties should be ordered width, background, border-bottom

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: src/app/components/jobs-slider/jobs-slider.component.scss [hunk 16]
```diff
@@ -0,0 +1,109 @@
...
+    color: $white;
+  }
+
+  .home-jobs-slider-show-all-jobs-link-container {
+    width: 100%;
+    display: flex;
+    flex-direction: column-reverse;
+
+    .home-jobs-slider-show-all-jobs-link {
+      cursor: pointer;
```
Comment (buren) [inline src/app/components/jobs-slider/jobs-slider.component.scss:28]: Properties should be ordered padding-left, cursor

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: src/app/components/jobs-slider/jobs-slider.component.scss [hunk 17]
```diff
@@ -0,0 +1,109 @@
...
+  .home-jobs-slider-new-jobs-title {
+    color: $color-secondary-gray;
+  }
+
+  .home-jobs-slider-user-jobs-title {
+    color: $white;
+  }
+
+  .home-jobs-slider-show-all-jobs-link-container {
+    width: 100%;
```
Comment (buren) [inline src/app/components/jobs-slider/jobs-slider.component.scss:23]: Properties should be ordered display, width, flex-direction

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

**Comment by buren** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 35 possible new issues (including those that may have been commented here).
* 9 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/justarrived/just-match-web/pulls/365.

File: src/app/components/user-job-card/user-job-card.component.scss [hunk 18]
```diff
@@ -0,0 +1,61 @@
...
+    color: $color-gray-200;
+  }
+}
+@media (max-width: $md-device-width) {
+  .home-job-item-container:first-of-type {
+    margin-left: 3em;
+  }
+
+  .home-job-item-container:last-of-type {
+    margin-right: 3em;
```
Comment (bellevik) [inline src/app/components/user-job-card/user-job-card.component.scss]: Change em to px.

File: src/app/components/job-card/job-card.component.scss [hunk 19]
```diff
@@ -0,0 +1,61 @@
...
+    color: $color-gray-200;
+  }
+}
+@media (max-width: $md-device-width) {
+  .home-job-item-container:first-of-type {
+    margin-left: 3em;
+  }
+
+  .home-job-item-container:last-of-type {
+    margin-right: 3em;
```
Comment (bellevik) [inline src/app/components/job-card/job-card.component.scss]: Change em to px.

File: src/app/components/jobs-slider/jobs-slider.component.scss [hunk 20]
```diff
@@ -0,0 +1,109 @@
+@import '../../styles/colors';
+@import '../../styles/dimensions';
+@import '../../styles/typography';
+$job-card-border-radius: 8px;
+
+.home-jobs-slider-container {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 3em;
```
Comment (bellevik) [inline src/app/components/jobs-slider/jobs-slider.component.scss]: Change em to px.

File: src/app/components/jobs-slider/jobs-slider.component.scss [hunk 21]
```diff
@@ -0,0 +1,109 @@
+@import '../../styles/colors';
+@import '../../styles/dimensions';
+@import '../../styles/typography';
+$job-card-border-radius: 8px;
+
+.home-jobs-slider-container {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 20px;
+  @media screen and (max-width: $md-device-width) {
```
Comment (buren) [inline src/app/components/jobs-slider/jobs-slider.component.scss:10]: @media declaration should be preceded by an empty line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#emptylinebetweenblocks)

File: src/app/components/job-card/job-card.component.scss [hunk 22]
```diff
@@ -0,0 +1,61 @@
...
+  .home-job-item-salary-will-perform-job-container {
+    background: $color-primary-cerise;
+    color: $white;
+  }
+
+  .home-job-item-salary-job-ended-container {
+    background: $color-gray-50;
+    color: $color-gray-200;
+  }
+}
```
Comment (buren) [inline src/app/components/job-card/job-card.component.scss:52]: Rule declaration should be followed by an empty line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#emptylinebetweenblocks)

File: src/app/components/job-card/job-card.component.scss [hunk 23]
```diff
@@ -0,0 +1,61 @@
...
+    text-align: center;
+  }
+
+  .home-job-item-salary-will-perform-job-container {
+    background: $color-primary-cerise;
+    color: $white;
+  }
+
+  .home-job-item-salary-job-ended-container {
+    background: $color-gray-50;
```
Comment (buren) [inline src/app/components/job-card/job-card.component.scss:49]: Properties should be ordered color, background

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: src/app/components/job-card/job-card.component.scss [hunk 24]
```diff
@@ -0,0 +1,61 @@
...
+    background: $color-primary-cerise;
+    padding: 10px;
+    font-weight: 500;
+    /* TODO: Jobs border radius variable */
+    border-radius: 0 0 $job-card-border-radius $job-card-border-radius;
+    text-align: center;
+  }
+
+  .home-job-item-salary-will-perform-job-container {
+    background: $color-primary-cerise;
```
Comment (buren) [inline src/app/components/job-card/job-card.component.scss:44]: Properties should be ordered color, background

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: src/app/components/job-card/job-card.component.scss [hunk 25]
```diff
@@ -0,0 +1,61 @@
...
+      @include truncate-owerflowing-text;
+    }
+  }
+
+  .home-job-item-salary-container {
+    color: $white;
+    background: $color-primary-cerise;
+    padding: 10px;
+    font-weight: 500;
+    /* TODO: Jobs border radius variable */
```
Comment (buren) [inline src/app/components/job-card/job-card.component.scss:38]: Use `//` comments everywhere

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#comment)

File: src/app/components/job-card/job-card.component.scss [hunk 26]
```diff
@@ -0,0 +1,61 @@
...
+
+    .home-job-item-name {
+      text-align: center;
+      white-space: normal;
+      @include truncate-owerflowing-text;
+    }
+  }
+
+  .home-job-item-salary-container {
+    color: $white;
```
Comment (buren) [inline src/app/components/job-card/job-card.component.scss:34]: Properties should be ordered padding, font-weight, color, text-align, background, border-radius

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: src/app/components/job-card/job-card.component.scss [hunk 27]
```diff
@@ -0,0 +1,61 @@
...
+    height: 110px;
+    padding: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    .home-job-item-name {
+      text-align: center;
+      white-space: normal;
+      @include truncate-owerflowing-text;
```
Comment (buren) [inline src/app/components/job-card/job-card.component.scss:29]: Expected item on line 29 to appear before line 27. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#declarationorder)

File: src/app/components/job-card/job-card.component.scss [hunk 28]
```diff
@@ -0,0 +1,61 @@
...
+  border-radius: $job-card-border-radius;
+  box-shadow: 0 2px 6px 0 rgba(0,0,0,0.4);
+  cursor: pointer;
+
+  &:hover {
+    transform: scale(1.02);
+  }
+
+  .home-job-item-name-container {
+    height: 110px;
```
Comment (buren) [inline src/app/components/job-card/job-card.component.scss:20]: Properties should be ordered display, height, padding, flex-direction

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: src/app/components/job-card/job-card.component.scss [hunk 29]
```diff
@@ -0,0 +1,61 @@
...
+@import '../../styles/typography';
+$job-card-border-radius: 8px;
+
+.home-job-item-container {
+  display: inline-block;
+  width: 180px;
+  margin: 5px;
+  background-color: $default-bg-color;
+  border-radius: $job-card-border-radius;
+  box-shadow: 0 2px 6px 0 rgba(0,0,0,0.4);
```
Comment (buren) [inline src/app/components/job-card/job-card.component.scss]: `0.4` should be written without a leading zero as `.4`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)

Comment (buren) [inline src/app/components/job-card/job-card.component.scss]: Commas in function arguments should be followed by a single space

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#spaceaftercomma)

Comment (buren) [inline src/app/components/job-card/job-card.component.scss]: Commas in function arguments should be followed by a single space

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#spaceaftercomma)

Comment (buren) [inline src/app/components/job-card/job-card.component.scss]: Commas in function arguments should be followed by a single space

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#spaceaftercomma)

File: src/app/components/job-card/job-card.component.scss [hunk 30]
```diff
@@ -0,0 +1,61 @@
+@import '../../styles/colors';
+@import '../../styles/dimensions';
+@import '../../styles/typography';
+$job-card-border-radius: 8px;
+
+.home-job-item-container {
+  display: inline-block;
+  width: 180px;
+  margin: 5px;
+  background-color: $default-bg-color;
```
Comment (buren) [inline src/app/components/job-card/job-card.component.scss:10]: Properties should be ordered display, width, margin, cursor, background-color, border-radius, box-shadow

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: src/app/components/job-card/job-card.component.scss (hunk 25 above)
Comment (buren) [inline src/app/components/job-card/job-card.component.scss:38]: TODO found

**Comment by buren** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 27 possible new issues (including those that may have been commented here).
* 9 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/justarrived/just-match-web/pulls/365.

**Comment by bellevik** [comment]
👍

**Comment by bex1** [comment]
This was reverted. Did not display jobs properly

**Pull request merged**